polaris-core/src/main/java/org/apache/polaris/core/persistence/bootstrap/SchemaOptions.java

```diff
@@ -31,6 +31,11 @@ public interface SchemaOptions {
   @Nullable
   String schemaFile();
 
+  @Value.Default
+  default boolean setupSchema() {
```

Comment:
I fell like this should go into application.properties instead. We are creating a second layer of configuration that is only accessible to the admin tool (I don't see how the server runtime can change this value).